src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,228 @@
+      <div className="flex items-center mt-10">
+        {isTopFetching ? (
+          <Spinner />
+        ) : topMondo ? (
+          <div
+            className="flex justify-center items-center gap-5"
+            onClick={() => {
+              setCurrentMondo(topMondo);
+              setMondoModal(true);
+              setIsTop(true);
+            }}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)